### ISSUE TITLE
blue screen with xrdp

### ISSUE THREAD

**Issue by pcs3rd** [neutrinolabs/xrdp#962]
after installing, with all files in right locations, xrdp comes up with a solid blue screen with a window claiming it starts to connect, then it fails and just says, "connection problem, giving up, some problem"

**Comment by metalefty** [comment]
Show us the log as well as #952.

**Comment by ErencanOzkan** [comment]
I have the same problem. i get blue screen for some time and get the same error, when i try to connect via microsoft remote desktop.

`connectiong to sesman ip 127.0.0.1 port 3350
sesman connect ok
sending login info to session manager, please wait...
login successful for display 10
starting connecting
connection problem, giving up
some problem`

Here is the xrdp.log file when i try to login.

`[20171211-10:09:45] [INFO ] starting xrdp with pid 6175
[20171211-10:09:45] [INFO ] listening to port 3389 on 0.0.0.0
[20171211-10:10:20] [INFO ] A connection received from: XXX.XXX.XXX.XXX port XXX
[20171211-10:10:20] [DEBUG] Closed socket 12 (AF_INET XXX.XXX.XXX.XXX:XXXX)
[20171211-10:10:20] [DEBUG] Closed socket 11 (AF_INET 0.0.0.0:3389)
[20171211-10:10:20] [DEBUG] Security layer: requested 11, selected 1
[20171211-10:10:21] [INFO ] TLS connection established from XXX.XXX.XXX.XXX port XXX: TLSv1.2 with cipher AES256-SHA
[20171211-10:10:21] [DEBUG] xrdp_000018e5_wm_login_mode_event_00000001
[20171211-10:10:21] [INFO ] Loading keymap file /etc/xrdp/km-00000409.ini
[20171211-10:10:21] [WARN ] local keymap file for 0x00000409 found and doesn't match built in keymap, using local keymap file
[20171211-10:10:32] [DEBUG] xrdp_wm_log_msg: connecting to sesman ip 127.0.0.1 port 3350
[20171211-10:10:32] [INFO ] xrdp_wm_log_msg: sesman connect ok
[20171211-10:10:32] [DEBUG] xrdp_wm_log_msg: sending login info to session manager, please wait...
[20171211-10:10:32] [DEBUG] return value from xrdp_mm_connect 0
[20171211-10:10:32] [INFO ] xrdp_wm_log_msg: login successful for display 10
[20171211-10:10:32] [DEBUG] xrdp_wm_log_msg: started connecting
[20171211-10:10:36] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:10:39] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:10:43] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:10:46] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:10:50] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:10:53] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:10:57] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:11:00] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:11:04] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:11:07] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:11:11] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:11:14] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:11:18] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:11:21] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:11:25] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:11:28] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:11:32] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:11:35] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:11:39] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:11:42] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:11:46] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:11:49] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:11:53] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:11:56] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:12:00] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:12:03] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:12:07] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:12:10] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:12:14] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:12:17] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:12:21] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:12:24] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:12:28] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:12:31] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:12:35] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:12:38] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:12:42] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:12:45] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:12:49] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:12:52] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:12:56] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:12:59] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:13:03] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:13:06] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:13:10] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:13:13] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:13:17] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:13:21] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:13:24] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:13:28] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:13:31] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:13:35] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:13:38] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:13:42] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:13:45] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:13:49] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:13:52] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:13:56] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:13:59] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:14:02] [DEBUG] xrdp_wm_log_msg: connection problem, giving up
[20171211-10:14:02] [DEBUG] Closed socket 19 (AF_UNIX)
[20171211-10:14:02] [DEBUG] xrdp_wm_log_msg: some problem
[20171211-10:14:02] [DEBUG] xrdp_mm_module_cleanup
[20171211-10:14:02] [DEBUG] Closed socket 18 (AF_INET 127.0.0.1:52686)`

**Comment by metalefty** [comment]
@ErencanOzkan Did you see #952?

**Comment by ErencanOzkan** [comment]
Yes i see [#952](https://github.com/neutrinolabs/xrdp/issues/952)
My log is different, first of all there is no [ERROR] log in my case.

**Comment by metalefty** [comment]
Okay, then, can you show me xrdp-sesman.log as well as #952?

**Comment by metalefty** [comment]
No feed back from the reporter more than 1 month, closing. I'll reopen this when you're back.

**Comment by ajeecai** [comment]
Exactly the same blue screen for me, I am installing from source - branch: devel, commit ae1514c1679e87edfdeba5ca3eb2b3659ef44b1a. Follow the instruction from https://github.com/neutrinolabs/xrdp/wiki/Building-on-Debian-8. 

Attached logs
[xrdp.log](https://github.com/neutrinolabs/xrdp/files/2486533/xrdp.log)
[xrdp-sesman.log](https://github.com/neutrinolabs/xrdp/files/2486538/xrdp-sesman.log)

**Comment by adocampo** [comment]
Sorry for necrobumping this old thread, but I'm facing the very same issue on manjaro, tried even with the very latest version of xrdp and xorgxrdp as for today (0.9.13).
xrdp.log
```
[20200311-11:00:09] [INFO ] Cannot find keymap file /etc/xrdp/km-0001040a.ini
[20200311-11:00:09] [INFO ] Loading keymap file /etc/xrdp/km-0000040a.ini
[20200311-11:00:09] [WARN ] local keymap file for 0x0001040a found and doesn't match built in keymap, using local keymap file
[20200311-11:00:13] [DEBUG] xrdp_wm_log_msg: connecting to sesman ip 127.0.0.1 port 3350
[20200311-11:00:13] [INFO ] xrdp_wm_log_msg: sesman connect ok
[20200311-11:00:13] [DEBUG] xrdp_wm_log_msg: sending login info to session manager, please wait...
[20200311-11:00:13] [DEBUG] return value from xrdp_mm_connect 0
[20200311-11:00:13] [INFO ] xrdp_wm_log_msg: login successful for display 10
[20200311-11:00:13] [DEBUG] xrdp_wm_log_msg: started connecting
[20200311-11:00:17] [DEBUG] Closed socket 19 (AF_UNIX)
[20200311-11:00:21] [DEBUG] Closed socket 19 (AF_UNIX)
[20200311-11:00:24] [DEBUG] Closed socket 19 (AF_UNIX)
...
[20200311-11:03:41] [DEBUG] Closed socket 19 (AF_UNIX)
[20200311-11:03:44] [DEBUG] xrdp_wm_log_msg: connection problem, giving up
[20200311-11:03:44] [DEBUG] Closed socket 19 (AF_UNIX)
[20200311-11:03:44] [DEBUG] xrdp_wm_log_msg: some problem
[20200311-11:03:44] [DEBUG] xrdp_mm_module_cleanup
[20200311-11:03:44] [DEBUG] Closed socket 18 (AF_INET 127.0.0.1:50424)
```
xrdp-sesman.log
```
[20200311-11:00:24] [ERROR] X server for display 10 startup timeout
[20200311-11:00:24] [ERROR] another Xserver might already be active on display 10 - see log
[20200311-11:00:24] [DEBUG] aborting connection...
[20200311-11:00:24] [CORE ] window manager (pid 121210) did exit, cleaning up session
[20200311-11:00:24] [INFO ] calling auth_stop_session and auth_end from pid 121209
[20200311-11:00:24] [DEBUG] cleanup_sockets:
[20200311-11:00:24] [DEBUG] cleanup_sockets: deleting /tmp/.xrdp/xrdp_chansrv_socket_10
[20200311-11:00:24] [DEBUG] cleanup_sockets: deleting /tmp/.xrdp/xrdpapi_10
[20200311-11:00:24] [DEBUG] cleanup_sockets: failed to delete /tmp/.xrdp/xrdpapi_10
[20200311-11:00:24] [INFO ] ++ terminated session:  username angel, display :10.0, session_pid 121209, ip 192.168.10.125:59748 - socket: 12
```

**Comment by aladin2000** [comment]
> Sorry for necrobumping this old thread, but I'm facing the very same issue on manjaro, tried even with the very latest version of xrdp and xorgxrdp as for today (0.9.13). xrdp.log
> 
> ```
> [20200311-11:00:09] [INFO ] Cannot find keymap file /etc/xrdp/km-0001040a.ini
> [20200311-11:00:09] [INFO ] Loading keymap file /etc/xrdp/km-0000040a.ini
> [20200311-11:00:09] [WARN ] local keymap file for 0x0001040a found and doesn't match built in keymap, using local keymap file
> [20200311-11:00:13] [DEBUG] xrdp_wm_log_msg: connecting to sesman ip 127.0.0.1 port 3350
> [20200311-11:00:13] [INFO ] xrdp_wm_log_msg: sesman connect ok
> [20200311-11:00:13] [DEBUG] xrdp_wm_log_msg: sending login info to session manager, please wait...
> [20200311-11:00:13] [DEBUG] return value from xrdp_mm_connect 0
> [20200311-11:00:13] [INFO ] xrdp_wm_log_msg: login successful for display 10
> [20200311-11:00:13] [DEBUG] xrdp_wm_log_msg: started connecting
> [20200311-11:00:17] [DEBUG] Closed socket 19 (AF_UNIX)
> [20200311-11:00:21] [DEBUG] Closed socket 19 (AF_UNIX)
> [20200311-11:00:24] [DEBUG] Closed socket 19 (AF_UNIX)
> ...
> [20200311-11:03:41] [DEBUG] Closed socket 19 (AF_UNIX)
> [20200311-11:03:44] [DEBUG] xrdp_wm_log_msg: connection problem, giving up
> [20200311-11:03:44] [DEBUG] Closed socket 19 (AF_UNIX)
> [20200311-11:03:44] [DEBUG] xrdp_wm_log_msg: some problem
> [20200311-11:03:44] [DEBUG] xrdp_mm_module_cleanup
> [20200311-11:03:44] [DEBUG] Closed socket 18 (AF_INET 127.0.0.1:50424)
> ```
> 
> xrdp-sesman.log
> 
> ```
> [20200311-11:00:24] [ERROR] X server for display 10 startup timeout
> [20200311-11:00:24] [ERROR] another Xserver might already be active on display 10 - see log
> [20200311-11:00:24] [DEBUG] aborting connection...
> [20200311-11:00:24] [CORE ] window manager (pid 121210) did exit, cleaning up session
> [20200311-11:00:24] [INFO ] calling auth_stop_session and auth_end from pid 121209
> [20200311-11:00:24] [DEBUG] cleanup_sockets:
> [20200311-11:00:24] [DEBUG] cleanup_sockets: deleting /tmp/.xrdp/xrdp_chansrv_socket_10
> [20200311-11:00:24] [DEBUG] cleanup_sockets: deleting /tmp/.xrdp/xrdpapi_10
> [20200311-11:00:24] [DEBUG] cleanup_sockets: failed to delete /tmp/.xrdp/xrdpapi_10
> [20200311-11:00:24] [INFO ] ++ terminated session:  username angel, display :10.0, session_pid 121209, ip 192.168.10.125:59748 - socket: 12
> ```

did you find a solution for that ?
